.github/workflows/dpu-server.yaml

```diff
@@ -0,0 +1,51 @@
+name: DPU server CI
```

Comment:
I suppose `UCC service` is more appropriate name for this product, isn't it?

Reply:
If yes, please reflect in the filenames as well

Reply:
The name of the file is dpu_server. So, it's ok.